gslb/test/ingestion/services_test.go

```diff
@@ -38,17 +35,7 @@ const (
 )
 
 func addGDPAndGSLBForSvc(t *testing.T) *gdpalphav2.GlobalDeploymentPolicy {
```

Comment:
Should we remove this function and just use addGDPAndGSLBForIngress?

Reply:
updated to directly use addGDPAndGSLBForIngress